crypto/rand/drbg_lib.c

```diff
@@ -403,6 +403,8 @@ static int drbg_status(void)
     int ret;
 
     CRYPTO_THREAD_write_lock(rand_drbg.lock);
+    if (parent->state == DRBG_UNINITIALISED)
+        RAND_DRBG_instantiate(rand_drbg, NULL, 0);
```

Comment:
yeah, forgot to push the updated commit.  just did so.  please reconfirm if you're okay with it.